src/test/java/com/bobocode/blyznytsia/bibernate/bootstrap/DefaultSessionFactoryBuilderTest.java

```diff
@@ -0,0 +1,17 @@
+package com.bobocode.blyznytsia.bibernate.bootstrap;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class DefaultSessionFactoryBuilderTest {
+
+  @Test
+  @DisplayName("Create session factory with datasource from default file")
```

Comment:
JFYI: you can use `@DisplayNameGeneration(CamelCaseNameGenerator.class)` to automatically generate human-readable test names out of method names.
But anyway I really liked that you spend time writing concise names for each test

Reply:
ok, thanks. But I prefer to have separate names for method itself and display name.